skimage/transform/_warps.py

```diff
@@ -129,8 +129,24 @@ def resize(image, output_shape, order=1, mode='reflect', cval=0, clip=True,
                 warn("Anti-aliasing standard deviation greater than zero but "
                      "not down-sampling along all axes")
 
+        # Translate modes used by np.pad to those used by ndi.gaussian_filter
+        if mode == "constant":
+            gf_mode = "constant"
+        elif mode == "edge":
+            gf_mode = "nearest"
+        elif mode == "symmetric":
+            gf_mode = "reflect"
+        elif mode == "reflect":
```

Comment:
@hmaarrfk what are you saying exactly? There's the SciPy convention and the NumPy convention, and for some crazy reason they are different. (I'm not sure whether we have a different one from both of those but I hope not!) I think we need this fixed in those projects, rather than here...

Reply:
Whoa mind blown!!!!! I’ll have to look into it again when it is time to hanger that part of my code